tests/test_templater.py

```diff
@@ -291,3 +293,46 @@ def test_set_template_verbosity():
     outcome = ''
     # Check that only the correct messages were logged
     assert result == outcome
+
+def test_log_decorator():
+    """Unit test for checking application of the logger decorator"""
+
+    input_file = os.path.join(uwtools_file_base, "fixtures/nml.IN")
+    config_file = os.path.join(uwtools_file_base, "fixtures/fruit_config.yaml")
+    logfile = os.path.join(os.path.dirname(templater.__file__), "templater.log")
+
+    outcome=\
+    """Finished setting up debug file logging in """ + logfile  + """
+Running script templater.py with args:"""
+
+    with tempfile.TemporaryDirectory(dir='.') as tmp_dir:
+        out_file = f'{tmp_dir}/test_render_from_yaml.nml'
+
+        args = [
+             '-i', input_file,
+             '-c', config_file,
+             '-o', out_file,
+             '-v'
+             ]
+
+        print("Test templater with logger decorator")
+        # Capture verbose stdout
+        outstring = io.StringIO()
+        with redirect_stdout(outstring):
+            templater.set_template(args)
+            print("Test calling templater logging object")
+            #loggers = [logging.getLogger(name) for name in logging.root.manager.loggerDict]
+            #print(f"Current loggers are: {loggers}")
+            log = logging.getLogger('uwtools.templater')
```

Comment:
@christinaholtNOAA I intended this to establish what we talked about on Friday as a check-in, then complete the implementation in a proper full PR.

It turns out this is all that is required to access the logging object, there's no need to try to force it to pass. Once its instantiated, the root manager can access it from the name. I added `name=__name__` to the initial Logger() call which will ensure the logger created by a method now carries that method's name and can be accessed as such, rather than leaving all of them as root. The commented out section was my verification of registered logger objects and the debug call proves that the called log is the existing one rather than a new one.